doc/command_ref.rst

```diff
@@ -1492,6 +1518,8 @@ The ``info`` subcommand lists description and summary information about packages
 Search Command
 --------------
 
+| Aliases: ``search``, ``se``
```

Comment:
Seems you've missed this one when separating out the main command...

Reply:
Thanks, fixed.